sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/util/ZipFileExtractor.java

```diff
@@ -9,7 +9,6 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
-import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperExitCode;
 import com.mercedesbenz.sechub.wrapper.xray.report.XrayWrapperReportException;
```

Comment:
We have a `ZipFileExtract` class (which makes absolute sense) but the zip mechanism is inside a complete other class which handles multiple IO things.

Maybe we should create a `ZipFileCreater` or `ZipFileArchiver` class for the zip mechanism?